tests/integration/test_views.py

```diff
@@ -91,3 +92,66 @@ def test_all_pages_for_single_page(page):
     page.save()
     assert qs.filter(live=False).exists()
     assert qs.filter(id=page.id).exists()
+
+
+@pytest.mark.django_db
+def test_sitemap_content_for_single_site(settings, site):
+    view = SitemapBuildableView()
+    view.request = view.create_request(view.sitemap_path)
+
+    content = view.get_content().decode()
+    expected_content = (
```

Comment:
This test looks fragile. Should we assume that the exact formatting of Django's sitemap generation will be consistent across versions? As an alternative we could parse the XML and check for the namespace, a single `<loc>` element inside a single `<url>` element etc.